docs/make.jl

```diff
@@ -47,8 +54,9 @@ makedocs(;
         prettyurls = get(ENV, "CI", "false") == "true",
         canonical = "https://deltares.github.io/Wflow.jl",
         assets = String[],
+		collapselevel = 2,
     ),
     pages = pages,
 )
 
-deploydocs(; repo = "github.com/Deltares/Wflow.jl", push_preview = true)
+#deploydocs(; repo = "github.com/Deltares/Wflow.jl", push_preview = true)
```

Comment:
Why is this commented out? Would be nice to see a preview. It will only deploy to the official docs when it is on the master branch.

Reply:
An artifact from my (local) testing, should be fixed now!